scripts/check_archived_pv

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Are you pretty hard set on the name? Not sure about others but I think I'd prefer something like `archiveStatus`

Reply:
I don't really care about the name, as long as it makes sense. `archiveStatus` is just as intuitive to me. 
There's an interesting mix of naming conventions used in this repo. Looks like I picked the wrong one. Is camel case the new standard in this repo?

Reply:
Yeah, there's an unfortunate number of conventions used and I'd be fine with camelCase, lowerwords, or lower_words. I had no reason behind picking archiveStatus, I think I'm just used to it as I've been looking at camViewer a lot recently.
In hindsight, I think I kinda prefer lowerwords for scripts/commands but it really doesn't matter. Maybe at some point we'll converge on one convention but it'll be a painful transition.

Reply:
I'll own camViewer and the only reason is that I wanted to avoid the same name as the underlying command. Could have gone with cam_viewer......

Reply:
Sooooo..... `archive_status` ? `archivestatus`? :)